.vscode/settings.json

```diff
@@ -13,6 +13,10 @@
     "python.linting.mypyEnabled": false,            // use the dmypy daemon extension instead - it's faster
     // "mypy.runUsingActiveInterpreter": true,      // in the devcontainer we override this to use an explicit path
     "autoDocstring.docstringFormat": "numpy",
+    "json.validate.enable": true,
+    "json.schemas": [
+        // TODO
```

Comment:
- [ ] Provides a way to automatically infer which schema to apply to the file when one isn't explicitly specified in itself.